SingularityUI/app/collections/TaskSearchResults.coffee

```diff
@@ -0,0 +1,63 @@
+Collection = require './collection'
+
+TaskSearchResult = require '../models/TaskSearchResult'
+
+class TaskSearchResults extends Collection
+
+    model: TaskSearchResult
+
+    params: ->
```

Comment:
because a number of our backbone collections and other classes have an @params of some sort, it might be easier to rename this method to something like `queryParams` or `getQueryParams`